diff -urN baobab-3.30.0.orig/data/org.gnome.baobab.desktop.in baobab-3.30.0/data/org.gnome.baobab.desktop.in
--- baobab-3.30.0.orig/data/org.gnome.baobab.desktop.in	2018-09-02 16:38:11.000000000 +0200
+++ baobab-3.30.0/data/org.gnome.baobab.desktop.in	2019-01-27 11:38:00.427295656 +0100
@@ -10,7 +10,6 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-MimeType=inode/directory;
 Categories=GTK;GNOME;System;Filesystem;X-GNOME-Utilities;
 NotShowIn=KDE;
 DBusActivatable=true
